frontend/lib/rh/rental-history.tsx

```diff
@@ -214,88 +226,94 @@ function RentalHistoryPreview(): JSX.Element {
         </p>
       </DemoDeploymentNote>
       <div className="field is-grouped jf-two-buttons">
-        <BackButton label="Back" to={JustfixRoutes.locale.rh.form} />
+        <BackButton to={JustfixRoutes.locale.rh.form} />
         <SessionUpdatingFormSubmitter
           mutation={RhSendEmailMutation}
           initialState={{}}
           onSuccessRedirect={JustfixRoutes.locale.rh.confirmation}
         >
           {(ctx) => (
-            <NextButton label="Submit request" isLoading={ctx.isLoading} />
+            <NextButton
+              label={li18n._(t`Submit request`)}
+              isLoading={ctx.isLoading}
+            />
           )}
         </SessionUpdatingFormSubmitter>
       </div>
     </Page>
   );
 }
 
+const KYR_LINKS: LocalizedOutboundLinkProps[] = [
+  {
+    children: <Trans>Met Council on Housing</Trans>,
+    hrefs: {
+      en: "https://www.metcouncilonhousing.org/help-answers/",
+    },
+  },
+  {
+    children: <Trans>Housing Court Answers</Trans>,
+    hrefs: {
+      en: "http://housingcourtanswers.org/glossary/",
+    },
+  },
+  {
+    children: <Trans>JustFix.nyc's Learning Center</Trans>,
+    hrefs: {
+      en:
+        "https://www.justfix.nyc/learn?utm_source=tenantplatform&utm_medium=rh",
```

Comment:
Make sure we file an issue to update this once we have the learning center localized to Spanish!

Reply:
Filed as #1648.